src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,21 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    private static final int MIN_AGE = 35;
+    private static final int MIN_PERIOD_IN_UKR = 10;
+    private static final String NATIONALITY = "Ukrainian";
+
+    @Override
+    public boolean test(Candidate candidate) {
+        int periodInUkr =
+                Integer.parseInt(candidate.getPeriodsInUkr().split("-")[1])
```

Comment:
```suggestion
                Integer.parseInt(candidate.getPeriodsInUkr().split(SPLITTER)[YEAR_TO])
```
this variables also better to make as constants